modules/virt-about-runstrategies-vms.adoc

```diff
@@ -50,16 +74,6 @@ The following table follows a VM's transition from different states. The first c
 
 [NOTE]
 ====
-In {VirtProductName} clusters installed using installer-provisioned infrastructure, when a node fails the MachineHealthCheck and becomes unavailable to the cluster, VMs with a RunStrategy of `Always` or `RerunOnFailure` are rescheduled on a new node.
+If a node in a cluster installed by using installer-provisioned infrastructure fails the MachineHealthCheck and is unavailable, VMs with `runStrategy: Always` or `runStrategy: RerunOnFailure` are rescheduled on a new node.
```

Comment:
Can this be:

```suggestion
If a node in a cluster installed by using installer-provisioned infrastructure fails the machine health check and is unavailable, VMs with `runStrategy: Always` or `runStrategy: RerunOnFailure` are rescheduled on a new node.
```

? Since usually the [object reference formatting](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting) would be saved for something like "the `MachineHealthCheck` resource", which would read kind of awkward here in context.